src/antares/craft/service/base_services.py

```diff
@@ -564,7 +564,7 @@ def config(self) -> BaseConfiguration:
         pass
 
     @abstractmethod
-    def update_study_settings(self, settings: StudySettings) -> Optional[StudySettings]:
+    def update_study_settings(self, settings: StudySettings) -> None:
```

Comment:
Here we could have a `StudySettignsUpdate` object with optionals.
But maybe you prefer to wait that we use the "function argument approach" before changing this ?
(where the update would take multiple named arguments)